<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaInfoUsuario</class>
 <widget class="QDialog" name="VentanaInfoUsuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>361</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>665</width>
    <height>361</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>665</width>
    <height>361</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Información del usuario</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../recursos/qtRecursosIconos.qrc">
    <normaloff>:/iconos/iconos/icono-app-dgtic.png</normaloff>:/iconos/iconos/icono-app-dgtic.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#VentanaInfoUsuario {
	background-color: #fff;
}

#frameGeneralInfoUsuario .QPushButton{
	color: #ffffff;
	background-color: #417fd2;
	border: 1px solid rgb(0, 0, 0);
	border-radius: 6px;
	padding-bottom: 2px;
	font: 8.5pt &quot;Segoe UI Black&quot;;
}

#frameGeneralInfoUsuario .QPushButton::hover{
	background-color: #5c8bc9;
}

#frameGeneralInfoUsuario .QPushButton::pressed{
	background-color: #063c85;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frameGeneralInfoUsuario">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
      <item>
       <widget class="QFrame" name="frameInfoUsuario">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>30</y>
           <width>411</width>
           <height>141</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="vlInfoUsuario">
          <item>
           <layout class="QHBoxLayout" name="hlInfoNombreUsuario">
            <item>
             <widget class="QLabel" name="labelInfoNombreUsuario">
              <property name="text">
               <string>Nombre de usuario:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="inputInfoNombreUsuario"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hlInfoTIpoRol">
            <item>
             <widget class="QLabel" name="labelInfoTipoRol">
              <property name="maximumSize">
               <size>
                <width>115</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Tipo de rol:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbInfoTipoRol">
              <item>
               <property name="text">
                <string>ADMINISTRADOR</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ESTANDAR</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hlInfoClaveUsuario">
            <item>
             <widget class="QLabel" name="labelInfoClaveUsuario">
              <property name="text">
               <string>Clave (Dejar vacío para no modificar):</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="inputInfoClaveUsuario"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameBotonesAccionUsuario">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>30</y>
           <width>371</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="hlBotonesAccionUsuario">
          <item>
           <widget class="QPushButton" name="botonAcualizarInfoUsuario">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Actualizar info</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonEliminarUsuario">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Eliminar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonCancelarInfoUsuario">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Cancelar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../recursos/qtRecursosIconos.qrc"/>
 </resources>
 <connections/>
</ui>
